HometaskPurchaseInStore/HWPurchaseInStore/App.cs

```diff
@@ -0,0 +1,64 @@
+﻿using HWPurchaseInStore.Models;
+using HWPurchaseInStore.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HWPurchaseInStore
+{
+    internal class App
+    {
+        public void RunApp()
+        {
+            GoodsInStore goodsInStore = new GoodsInStore();
+            SelectGoods selectGoods = new SelectGoods();
+            GetClientsData getData = new GetClientsData();
+            PlaceOrder placeOrder = new PlaceOrder();
+
+            Console.WriteLine("In our store we have next goods:");
+
+            foreach (string value in goodsInStore._fruits)
+            {
+                Console.Write(value + " ");
+            }
+            Console.WriteLine("\n \nPlease, choose up to 10 units 1-by-1 by writing exact names of fruits then write \"finish\":\n");
+            selectGoods.AddFruit();
+
+            if (!string.IsNullOrEmpty(selectGoods._basket[0]))
+            {
+                getData.GetClient();
+                // Console.WriteLine($"{getData._firstName}");
+                Console.WriteLine("\nYou have chosen products, type \"yes\" to finish your order or \"no\" to end program");
+                string command = Console.ReadLine();
+                if (command != "yes" && command != "no")
+                {
+                    Console.WriteLine("WRITE IT CORRECTLY !!!!!!!!!!!!!!!!!!!!!!!!!!!! The program need to restart");
+                }
+                else if (command == "no")
```

Comment:
You can use: command .ToLower(); In this case if I write Yes, YES, yES to console, your method will not be broken